interface/super/edit_layout.php

```diff
@@ -988,19 +969,27 @@ function writeFieldLine($linedata)
 <?php html_header_show();?>
 
 <!-- supporting javascript code -->
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-2-2/index.js"></script>
-<script type="text/javascript" src="../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
-
 <link rel="stylesheet" href='<?php  echo $css_header ?>' type='text/css'>
+<link  href="<?php echo $GLOBALS['assets_static_relative']; ?>/select2-4-0/dist/css/select2.min.css" rel="stylesheet" />
+<link href="<?php echo $GLOBALS['assets_static_relative']; ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css" />
+
+<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-3-1-1/index.js"></script>
+<script src="<?php echo $GLOBALS['assets_static_relative']; ?>/bootstrap-3-3-4/dist/js/bootstrap.min.js" type="text/javascript"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/select2-4-0/dist/js/select2.full.js"></script>
+<script type="text/javascript" src="../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
+<!-- Below for future modifier dialog use -->
+<!-- <link rel="stylesheet" href="<?php //echo $GLOBALS['assets_static_relative']; ?>/datatables.net-bs-1-10-13/css/dataTables.bootstrap.min.css" type="text/css">
+<script type="text/javascript" src="<?php //echo $GLOBALS['assets_static_relative']; ?>/datatables.net-1-10-13/js/jquery.dataTables.min.js"></script>
+<script type="text/javascript" src="<?php //echo $GLOBALS['assets_static_relative']; ?>/datatables.net-bs-1-10-13/js/dataTables.bootstrap.min.js"></script> -->
```

Comment:
rec removing all above lines of code below <!-- supporting javascript code --> and doing the following:
1. add this here `Header::setupHeader('select2');`
2. add the following above at line 24 (below the require_once stuff) `use OpenEMR\Core\Header;`

Reply:
I have a problem with the Header class because it forces the OpenEMR styling theme on the script. Much of that theme uses !important which in most cases overrides many bootstrap styling.  Now I get why Robert had to make many compromises trying to restyle app and it is a monstrous effort but when trying to restyle by hand, problems occur. I don't want to touch the core stylings for fear what I think is a fix may be important and Robert had a reason for doing it that way. Whew, I probably made a mess of this explanation.

Reply:
@sjpadgett , Using Header here should change nothing from what you have. Note $css_header is already included here which brings in core etc. The only actual difference will be that select2 also brings in this styling (which appears to be a standard select2 styling when used with bootstrap): https://github.com/openemr/openemr/blob/master/config/config.yaml#L78

Reply:
Okay, done and testing...